README.md

```diff
@@ -4,11 +4,11 @@
 
 ## Homepage
 
-![freesound-player's homepage as of 7:26 PM PST Thursday, August 13, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/current-appearance-of-freesound-player's-homepage-2020-08-13-192528.jpg)
+![freesound-player's homepage as of 7:49 PM PST Tuesday, September 22, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/homepage-2020-09-23-at-6.05.05-PM.png)
 
 ## After Selecting Sound
 
-![freesound-player's appearance after selecting a sound as of 7:26 PM PST Thursday, August 13, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/appearance-of-freesound-player-after-selecting-sound-2020-08-13-192709.jpg)
+![freesound-player's appearance after selecting a sound as of 7:55 PM PST Tuesday, September 22, 2020](https://github.com/amilajack/freesound-player/blob/master/screenshots/soundpage-2020-09-23-at-6.07.14-PM.png)
```

Comment:
```suggestion
![freesound-player's appearance after selecting a sound as of 7:55 PM PST Tuesday, September 22, 2020](screenshots/soundpage-2020-09-23-at-6.07.14-PM.png)
```

Reply:
I'll make the same change here, as well. I appreciate the tip!

Reply:
<img width="570" alt="Screen Shot 2020-09-24 at 5 05 59 PM" src="https://user-images.githubusercontent.com/6374832/94212140-3962b900-fe88-11ea-9c33-bbc77864012d.png">
Also I'm using GH's suggestions feature! Try it out, its super cool!

Reply:
See the "commit suggestion" dropdown

Reply:
I did! It lets me make the commit you suggested without having to manually make the change myself. It's really neat!